common/Tests/Framework/src/AITestClass.ts

```diff
@@ -683,6 +731,32 @@ export class AITestClass {
         }
     }
 
+    protected _assertEventsRemoved() {
+        this._assertNoEvents(window, "window");
+        this._assertNoEvents(document, "document");
+        if (document["body"]) {
+            this._assertNoEvents(document["body"], "body");
+        }
+
+        if (navigator) {
+            this._assertNoEvents(navigator, "navigator");
+        }
+
+        if (history) {
+            this._assertNoEvents(history, "history");
+        }
+    }
+
+    protected _assertHooksRemoved() {
```

Comment:
This is the current set of objects that will be validated -- these will grow as we find plugins that are adding hooks.